.travis.yml

```diff
@@ -21,9 +21,10 @@ install:
 
   - conda env create -n test-environment -f environment.yml
   - source activate test-environment
-  - python setup.py install
+  - pip install -e .
+  - pip install -e .[dev]
   
-script: py.test --cov=aspire tests
+script: pytest --random-order --cov=aspire tests
```

Comment:
We really should be using the environment variables when we can, instead of flags. This provides the most flexibility, in case we decide to drop this dependency in the future. Especially because it is so easy to do in most cases:
```
PYTEST_ADDOPTS=--random-order pytest --cov=aspire tests
```

The above form will work whether or not we have `pytest-random-order` installed.
Note that using the above environment variable will only add to the options, not override them, so whatever is in `pytest.ini` will still be considered (you may want to verify this).

Reply:
Since environment variables in Windows can be tricky, this may be a good time to investigate the `variables` section in an Azure pipeline which should allow you to pass them on to Linux/OSX/Windows without having to do anything platform-specific.

https://docs.microsoft.com/en-us/azure/devops/pipelines/process/variables

Reply:
Edit: Never mind - an unrecognized value of `PYTEST_ADDOPTS` is not ignored, but crashes. So there's no advantage to using it over the `--random-order` flag. You're fine.

Reply:
Thanks. I will keep using the option directly.